### PR TITLE
V5

### PR DESCRIPTION
**Pull request by ihadeed**

### CODE REVIEW

**Comment by melkir** [comment]
There is half of the changes that are only related to the code style indent or import order.
Is it intended ?

**Comment by ihadeed** [comment]
The class definition order is intentional.

Style changes (indention..etc) was probably my IDE ensuring everything
matches .editorconfig

**Comment by joshbabb** [comment]
Nice, need testing?

File: src/@ionic-native/plugins/device-feedback/index.ts [hunk 1]
```diff
@@ -1,6 +1,14 @@
 import { Injectable } from '@angular/core';
 import { Cordova, IonicNativePlugin, Plugin } from '@ionic-native/core';
 
+export interface DeviceFeedbackEnabled {
```
Comment (ihadeed) [inline src/@ionic-native/plugins/device-feedback/index.ts]: Should be called `DeviceFeedbackStatus` @danielcregg

Comment (ihadeed) [inline src/@ionic-native/plugins/device-feedback/index.ts]: Should be called `DeviceFeedbackStatus` @danielsogl

Comment (danielsogl) [reply src/@ionic-native/plugins/device-feedback/index.ts]: fixed

**Comment by wKoza** [comment]
Hi @ihadeed,
When do you plan to release ionic-native v5 final ?

**Comment by timbru31** [comment]
The git tags for newer releases of the beta are missing, too. Can you add them please?

**Comment by wf9a5m75** [comment]
Is the v5 finally ready to merge?

**Comment by danielsogl** [comment]
We will release V5 together with Ionic v4 because of Angular 6. V5 is not backwards compatible.

**Comment by wf9a5m75** [comment]
Got it. Thanks.

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,6 +12,107 @@
 
 
 
+<a name="5.0.0-beta.14"></a>
```
Comment (wf9a5m75) [inline CHANGELOG.md]: Did you revert to v5.0.0-beta.14, or you just copy & past by mistake?

**Pull request merged**